### DISCUSSION TITLE
求助，关于alist挂载阿里云盘视频卡顿

### DISCUSSION THREAD

**Discussion by lovezongzi** [AlistGo/alist#5601] [General]
求助，关于alist挂在的阿里云盘视频，最近总是看二十分钟左右就卡顿，必须刷新页面才好，20分钟之后又会出现这种情况，有人知道是什么原因吗？
![image](https://github.com/alist-org/alist/assets/75051617/a6c4f922-113e-41ed-9bb1-231726791ccc)

**Comment by aru213** [comment]
遇到一样的问题，还没找到解决方法

**Reply by yanzhang4013** [reply]
我的是15分钟，重新刷新页面就好了。

**Reply by lovezongzi** [reply]
我进行了详细测试，是15分钟，每个人或多或少但是基本都是15分钟，原因已经知道了，是阿里那边将时间戳改成了15分钟，但是目前我没有找到可以解决的办法，只能等开发者进行修复了。多给一些时间吧。如果有好的解决办法可以分享一下

**Reply by Deepseaon** [reply]
已经修复了更新到3.30可以自动刷新视频链接，只会卡一下就好

**Comment by citylyz** [comment]
用网页界面的阿里云盘默认播放器aliyun video previewer，没有限速，如果选video模式，就是限速了，只有500KB。感觉是用非阿里云播放器限速了。

**Reply by lovezongzi** [reply]
原因已经知道了，是阿里那边将时间戳改成了15分钟，但是目前我没有找到可以解决的办法，只能等开发者进行修复了。我换成了video视频无法加载，安稳大佬说，可能是编码问题，我也不太懂，多给一些时间吧。如果有好的解决办法可以分享一下

**Comment by Silent-wqh** [comment]
苹果端的infuse支持直接挂载阿里云盘，但是不存在限速。与此同时，我用infuse通过webdav连接alist挂载的阿里云盘，限速500KB。

**Comment by CardiacCycle** [comment]
webdav挂载阿里云盘open遇到了同样的问题；同时，被限速到4M/s左右；webdav挂载中国移动云盘播放正常

**Reply by tonsen911** [reply]
不是所有片都4M吧，我的是看4k 4m，普通的能到10几

**Reply by zhoutao-cn** [reply]
实锤了，不是 WEBDAV 的问题，是阿里云针对 Allist 限速了。换用其它 TOKEN 及 Oauth令牌链接 又可以享受满速了。

**Reply by kaixinxiaowangzi666** [reply]
分享一下谢谢

**Reply by LIUBANGJIAN** [reply]
请问如何更换换用其它 TOKEN 及 Oauth令牌链接呢？

**Comment by MicroMatrixOrg** [comment]
> 实锤了，不是 WEBDAV 的问题，是阿里云针对 Allist 限速了。换用其它 TOKEN 及 Oauth令牌链接 又可以享受满速了。
有没有文档教程啊。我找到个方法。但是没法设置默认播放器。
![image](https://github.com/alist-org/alist/assets/21361701/94f75af3-1490-4177-adbf-edeb5eaa2d17)

**Comment by KnIfER** [comment]
怎么换其他令牌啊

**Comment by Deepseaon** [comment]
已经修复了更新到3.30可以自动刷新视频链接，只会卡一下就好

**Reply by apanoo** [reply]
每10来分钟自动刷新页面接上进度的这个方案，体验有点不太好啊😂😂😂特别是移动端全屏的情况下

**Reply by Deepseaon** [reply]
是这样的，每次都是突然开闪，不过也是无奈之举了，阿里做了限制只能刷新连接，看看后面作者能不能优化的平滑一点吧，如果是卡住转一下圈再播放应该能舒服好多

**Reply by apanoo** [reply]
目前这个调整问题还是不少，比如暂停视频的时候计时器还是在跑，超过14分钟自动就开始播放了，除此之外各种小问题估计也有。不知道是否可以在后端做个m3u8代理，刷新url的事交给后端
